src/main/kotlin/org/jitsi/nlj/transform/node/incoming/VideoParser.kt

```diff
@@ -51,11 +54,14 @@ class VideoParser(
                     val vp8Packet = rtpPacket.toOtherType(::Vp8Packet)
                     findRtpEncodingDesc(vp8Packet)?.let {
                         vp8Packet.qualityIndex = it.index
+                        vp8Packet
```

Comment:
This lets non-VP8 packets without a signaled SSRC pass. I think we should drop them

Reply:
If we want to filter based on SSRC, we can do that in a place agnostic to audio/video.

Reply:
We are filtering based on SSRC (it's just implicit in `findRtpEncodingDesc`). I don't think we need to do the same for audio because we don't do anything special with it (and we don't even keep `tracks` objects for it). I think we should do it for all video, because it is likely to cause the same problem we see with VP8: seemingly unrelated logs (or exceptions) in the `wants/send` code.

Reply:
Ok, want me to do that as part of this PR?